packages/lambdas/src/tenovi.ts

```diff
@@ -0,0 +1,71 @@
+import { getSecret } from "@aws-lambda-powertools/parameters/secrets";
+import * as Sentry from "@sentry/serverless";
+import { APIGatewayEvent } from "aws-lambda";
+import axios from "axios";
+import status from "http-status";
+import { capture } from "./shared/capture";
+import { getEnvOrFail } from "./shared/env";
+
+// Keep this as early on the file as possible
+capture.init();
+
+const apiServerURL = getEnvOrFail("API_URL");
+const tenoviAuthHeader = getEnvOrFail("TENOVI_AUTH_HEADER");
+
+const api = axios.create();
+
+const buildResponse = (status: number, body?: unknown) => ({
+  statusCode: status,
+  body,
+});
+
+const defaultResponse = () => buildResponse(status.NO_CONTENT);
+
+export const handler = Sentry.AWSLambda.wrapHandler(async (event: APIGatewayEvent) => {
+  if (!event.body) {
+    console.log("Event has no body - will not be forwarded to the API");
+    return defaultResponse();
+  }
+
+  const authHeader: string = (await getSecret(tenoviAuthHeader)) as string;
+  if (!authHeader) {
+    throw new Error(`Config error - TENOVI_AUTH_HEADER doesn't exist`);
```

Comment:
This error message leads me to think the env var is missing, which is not the case here. Maybe indicate that the secret was not found or something?